mantis-control-plane/mantis-control-plane-server/build.gradle

```diff
@@ -31,6 +31,7 @@ ext {
 dependencies {
     api project(':mantis-control-plane:mantis-control-plane-core')
 
+    api "com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.11.2"
```

Comment:
I think this dep should be shaded in mantis-shaded and used through that. Are there reasons why we don't want to shade this?

Reply:
I think the import is using mantis-shaded already so I think there is no point in adding this.